cmd/juju/cloud/updatecredential.go

```diff
@@ -389,9 +412,9 @@ func (c *updateCredentialCommand) updateRemoteCredentials(ctx *cmd.Context, upda
 			continue
 		}
 		ctx.Infof(`
-Controller credential %q for user %q on cloud %q updated.
+Controller credential %q for user %q on cloud %q %ved.
```

Comment:
updateed? deleteed? createed?

Reply:
No, this is correct and caters or things like 'added' and 'loaded". so when 'updated' is desired, we'd pass in "updat". Anyway due to some rules and exceptions in English language, there are plenty of places where string concatenation and modification can catch people unawares. This is why there are tests, eye balls, etc. Here, is one of these awesome places. 
Having said that though, it's cleaner if a full correct verb in correct tense is passed in, so I'll adjust the code accordingly.